packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue

```diff
@@ -76,6 +89,11 @@ const panel = ref();
 
 const togglePanel = (event) => panel.value.toggle(event);
 
+const getProjectName = () => {
+	const projectName = useProjects().activeProject.value?.name;
+	return projectName ?? '';
+};
```

Comment:
This should `activeProject` should never return `null` it should return an empty string, to avoid making those those test here.